config/environments/test.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.configure do
```

Comment:
I'm pretty sure this is the wrong approach here and that this configuration should only happen for the one railtie spec.  Yes?? 

However, it's getting late and I gotta jet, so leaving this question here.